eslint.json

```diff
@@ -0,0 +1,24 @@
+{
+	"extends": "eslint:recommended",
```

Comment:
jQuery projects are not supposed to use this preset but the jQuery one. This is to me a prerequisite to the migration, we can't migrate to a preset that doesn't verify rules jQuery cares about.

Reply:
which preset would you recommend then?

Reply:
The official jQuery one: https://www.npmjs.com/package/eslint-config-jquery